src/App.js

```diff
@@ -1,24 +1,117 @@
 import './App.css';
+import React, { useState, useEffect } from "react";
+import {MdClear, MdAddCircle, MdCheckCircle} from 'react-icons/md'
+import{v4 as UuidV4} from "uuid";
 
-function App() {
+const App = () => {
+  /* 로컬 스토리지 저장 */
+  const initState = JSON.parse(localStorage.getItem("todos")) || [];
+  const [input, setInput] = useState("");
+  const [todos, setTodos] = useState(initState);
+  const [dones, setDones] = useState([]);
+
+  useEffect(()=>{
+    localStorage.setItem("todos", JSON.stringify(todos));
+  }, [todos]);
+
+/* todo length를 통해 아이템 카운트*/
+  return <div className='Background'>
+  <div className='do-list'>아직 완료 안된 일 ({todos.length})</div>
+  <div className='box-list'>
+  <UnDoneList todos={todos} setTodos = {setTodos}/>
+  </div>
+  <div className='do-list'>완료된 일 ({dones.length}) </div>
+  <div className='box-list'>
```

Comment:
https://ko.reactjs.org/docs/introducing-jsx.html
![image](https://user-images.githubusercontent.com/50395394/135741149-492c2a22-89ca-433a-911a-0cad910d9cdb.png)

className 명명을 lowerCamelCase 규칙을 사용하면 좋을 것 같아요!


```suggestion
  return <div className='background'>
  <div className='doList'>아직 완료 안된 일 ({todos.length})</div>
  <div className='boxList'>
  <UnDoneList todos={todos} setTodos = {setTodos}/>
  </div>
  <div className='doList'>완료된 일 ({dones.length}) </div>
  <div className='boxList'>
```